be/src/storage/tablet.cpp

```diff
@@ -1052,6 +1053,7 @@ void Tablet::build_tablet_report_info(TTabletInfo* tablet_info) {
     tablet_info->__set_path_hash(_data_dir->path_hash());
     tablet_info->__set_is_in_memory(_tablet_meta->tablet_schema().is_in_memory());
     tablet_info->__set_enable_persistent_index(_tablet_meta->get_enable_persistent_index());
+    tablet_info->__set_binlog_version(_tablet_meta->get_binlog_config()->version);
```

Comment:
What if not exists?

Reply:
there is a default configuration when creating a table, even if binlog is disable